0-bootstrap/modules/jenkins/main.tf

```
@@ -0,0 +1,117 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  cicd_project_name           = format("%s-%s", var.project_prefix, "cicd")
+  impersonation_enabled_count = var.sa_enable_impersonation == true ? 1 : 0
```

nit: `sa_enable_impersonation` is a bool so we can just do
```suggestion
  impersonation_enabled_count = var.sa_enable_impersonation ? 1 : 0
```

---

Done